paparazzi/paparazzi-gradle-plugin/src/test/projects/lifecycle-usages/build.gradle

```diff
@@ -22,4 +22,5 @@ android {
 
 dependencies {
   testImplementation 'androidx.activity:activity-compose:1.6.1'
+  testImplementation libs.truth
```

Comment:
can the assertions part of these tests be moved out of the plugin integration tests and into Paparazzi tests?

oh wait nvm, probably not, cuz androidx savedstate and friends are aars, and not jars....drat.